test/Api.Test/SecretsManager/Controllers/RequestSMAccessControllerTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿#nullable enable
```

Comment:
It is encouraged to use `#nullable enable` on new features, but not required. 

If we are going to use it on this feature, we should have it on the source files for the command, controller, etc.

If not, I would recommend taking it out of this test file.

Reply:
thank you!